config/webpack.config.js

```diff
@@ -14,12 +16,15 @@ var production =
   process.env.RAILS_ENV === 'production' ||
   process.env.NODE_ENV === 'production';
 
-var config = {
-  entry: {
-    // Sources are expected to live in $app_root/webpack
-    bundle: './webpack/assets/javascripts/bundle.js'
-  },
+var plugins = JSON.parse(execSync('./script/plugin_webpack_directories.rb'));
 
+var config = {
+  entry: Object.assign(
+    {
+      bundle: './webpack/assets/javascripts/bundle.js'
+    },
+    plugins.entries
```

Comment:
@ohadlevy The agent that executes the roles can run on the proxy or on Foreman itself. In fact it's the first plugin that introduced this "_core" gem dependency concept to allow reusing the code to run actions in the proxy or in Foreman.